plugin.video.videodevil/resources/catcher.list

```diff
@@ -102,17 +102,20 @@ build=%s|Referer=https://www.youporn.com/
 quality=standard
 ########################################################
 title=extremetube.com
-target=quality_480p":"(http[^"]+)
+target=quality_(?:180|240)p":"([^"]+)
 url=%s
 header=Cookie|age_verified=1
-quality=standard
-target=quality_720p":"(http[^"]+)
+action=prefix
+quality=low
+target=quality_(?:360|480)p":"([^"]+)
 url=%s
 header=Cookie|age_verified=1
-quality=high
-target=quality_1080p":"(http[^"]+)
+action=prefix
+quality=standard
+target=quality_(?:720|1080)p":"([^"]+)
 url=%s
 header=Cookie|age_verified=1
+action=prefix

```

Comment:
Why do you need this  action ? Why not just use url=https:%s ?

Reply:
This is done intentional because the spankcdn urls return with https://blah sometimes and //blah some other times.
using url=https:%s breaks whenever the returned url has https  already. So the action adds https only if https already not there.